test/unit/metrics/test_group_sklearn_wrappers.py

```diff
@@ -36,7 +36,8 @@
 supported_metrics_weighted_binary = [(skm.precision_score, metrics.group_precision_score),
                                      (skm.recall_score, metrics.group_recall_score),
                                      (skm.roc_auc_score, metrics.group_roc_auc_score),
-                                     (skm.mean_squared_error, metrics.group_mean_squared_error)]
+                                     (skm.mean_squared_error, metrics.group_mean_squared_error),
+                                     (skm.r2_score, metrics.group_r2_score)]
```

Comment:
I have a feeling that 'binary' isn't a good name for this variable...

Also, what about `group_root_mean_squared_error` ?